docs/modeling/model-sets.rst

```diff
@@ -0,0 +1,61 @@
+.. include:: links.inc
+
+Model sets
+==========
+
+In some cases it is necessary to describe many models of the same type but with
+different sets of parameter values.  This could be done simply by instantiating
+as many instances of a `~astropy.modeling.Model` as are needed.  But that can
+be inefficient for a large number of models.  To that end, all model classes in
```

Comment:
This section might benefit from a more specific example of an application that would benefit significantly from the Model Set machinery.

Reply:
I agree.  And it should be in a separate PR that focuses on model sets.